### DISCUSSION TITLE
Ventoy File Manager(VentoyFM)

### DISCUSSION THREAD

**Discussion by crasadure** [ventoy/Ventoy#1301] [Ideas]
hi @ventoy 
I am following your project development and your forum web site..
https://forums.ventoy.net/showthread.php?tid=1904
The user is "nguyen ha thai trong" was developed a ISO file **ventoyfm_grubfm.iso** ...I tested it, some times it is gaving some errors. so it is not yet suitable  for ventoy.
May be you can add this future to Ventoy multiboot. Users can select  ISO and WIM files from other internal and external disks.
İmage1:
![ventoyfm1](https://user-images.githubusercontent.com/54566818/144725589-f81fc4c0-8439-4962-8855-c47b8b3e0992.jpg)
İmage2:
![ventoyfm2](https://user-images.githubusercontent.com/54566818/144725595-99c89592-e7b7-4338-99a3-325caeda1160.jpg)
İmage3:
![ventoyfm3](https://user-images.githubusercontent.com/54566818/144725598-26c8f8fc-bd4a-4f53-9cd8-1f5930c3e272.jpg)

I captured a video about my suggestion,pls watch my short video..
Video: https://vimeo.com/653277114

Can you add this  future to Ventoy multiboot system?

**Comment by ventoy** [comment]
NO.
The disk layout is part of Ventoy design. So Ventoy can only work well in where it installed.
May be it can work with some Windows or Linux Live ISO, but it's not official and can not boot all the ISO files supported by official Ventoy.
Ventoy need to completely rewrite for work like grubfile manager. I will not do this.
